docker/observability/otel/otel-collector-config.yml

```diff
@@ -25,6 +25,12 @@ processors:
   memory_limiter:
     check_interval: 1s
     limit_mib: 1024
+  transform:
+    metric_statements:
+      - context: datapoint
+        statements:
+        - set(attributes["client"], resource.attributes["of.client"])
+        - set(attributes["target"], resource.attributes["of.target"])
```

Comment:
Could you please explain why this is needed? I though "client" and "target" attributes are already set during deployment? Doesn't prometheus see it the same way as loki?

Reply:
Thank you for the explanation!